.github/workflows/continuous_integration.yaml

```
@@ -67,6 +70,12 @@ jobs:
           cargo test --workspace --locked --target ${{ matrix.target }} --all-features --all-targets
           -- --nocapture --quiet
 
+      - name: Test workspace with all feature combinations
+        uses: actions-rs/cargo@v1
+        with:
+          command: all-features
+          args: test --workspace
```

```suggestion
        run: cargo all-features test --workspace
```
If we want to avoid `actions-rs/cargo@v1`, this should work.